prime-router/metadata/organizations-local.yml

```diff
@@ -16,7 +16,8 @@
       - name: elr
         topic: covid-19
         schema: az/az-covid-19
-        jurisdictionalFilter: { standard.testing_lab_state: AZ }
+        jurisdictionalFilter:
+          - matches(standard.ordering_facility_state,AZ)
```

Comment:
Throughout the organization.yml files, the new `matches(column, value) `replaces the old   `column: value` syntax.